Packs/VersaDirector/Integrations/VersaDirector/test_data/input_data.py

```diff
@@ -0,0 +1,316 @@
+# HEADING: --------------- COMMAND FUNCTIONS TESTS ARGUMENTS ---------------
+
+# Test function: test_get_access_token
+# Arguments: (access_token_request_return_value, auth_credentials_request_return_value, expected_output)
+case_valid_arguments = (
+    {"access_token": "123456"},
+    {"client_id": "1234", "client_secret": "abcde1234"},
+    "To complete the Auth process, please copy the following values:\nclient_id: 1234\nclient_secret: abcde1234\nand use them as arguments to run command:'vd-auth-complete'",
+)
+get_access_token_args = [case_valid_arguments]
```

Comment:
If there is only one case, no need to use parametrize - just inline those values. Same in other places.

Reply:
I see you moved the values to the other file but kept the `pytest.mark.parametrize` - that is not what I meant. We are using parametrize when there is more than one case that we want to check. Here, we only have one case and do not need the parametrize- we can just use the values themselves in the test function.